### PR TITLE
Grammer polishing for the README

### PR DESCRIPTION
**Pull request by jsoref**

### CODE REVIEW

**Comment by ChristianSi** [comment]
"MIME type" is correct, "MIME-type" is not, see [Media type](https://en.wikipedia.org/wiki/Media_type).

Otherwise, this looks good to me!

However, there is a tiny amount of overlap with my (still unmerged) clean-up PR #732, so it might make sense to wait until that one is merged and then adjust this one as needed.

**Comment by jsoref** [comment]
Sorry, I should know better. I've reported the incorrect suggestion to app.gammarly.com.

I've tried to trigger a notification for the other PR. Hopefully that'll be enough for me to know when to update this. I presume you're OK w/ a rebased commit.

File: README.md [hunk 1]
```diff
@@ -993,7 +993,7 @@ TOML files should use the extension `.toml`.
 MIME Type
 ---------
 
-When transferring TOML files over the internet, the appropriate MIME type is
+When transferring TOML files over the internet, the appropriate MIME-type is
```
Comment (pradyunsg) [inline README.md:996]:
```suggestion
When transferring TOML files over the internet, the appropriate MIME type is
```

**Comment by jsoref** [comment]
@pradyunsg: do you folks prefer extra commits or a squash-rebase?

(I'm assuming I'm waiting on the other PR to merge...)

**Comment by pradyunsg** [comment]
No preference tbh. Anything works. I'll likely end up squash-merging, so... 🤷‍♂️

**Comment by pradyunsg** [comment]
Alrighty! #732 is merged. Now, this PR has merge conflicts. :)

**Comment by jsoref** [comment]
@pradyunsg: ok, now you can squash merge :-)

File: README.md [hunk 2]
```diff
@@ -157,7 +157,7 @@ first = "Tom" last = "Preston-Werner" # INVALID
 Keys
 ----
 
-A key may be either bare, quoted or dotted.
+A key may be either bare, quoted, or dotted.
```
Comment (pradyunsg) [inline README.md:160]: Hmmm... I'm gonna let this slide through.

Comment (jsoref) [reply README.md:160]: I checked before making that change, the document appears to favor Oxford commas. I don't usually go off randomly changing between styles. (My goal is to get things merged, not cause style fights.)

**Comment by pradyunsg** [comment]
Thanks for the PR @jsoref! ^>^

**Pull request merged**